### ISSUE TITLE
Updating to v1.3.1 breaks @apply usages for `thumb` classes

### ISSUE THREAD

**Issue by ConnorJamesLow** [adoxography/tailwind-scrollbar#25]
On v1.0.2, this works:

```css
.scroll-y {
  @apply overflow-x-hidden overflow-y-scroll scrollbar-thin scrollbar-thumb-green-300 
    hover:scrollbar-thumb-purple-500;
}
```

But once I updated to v1.3.1, the `scrollbar-thumb-green-300` and `...-purple-500` classes don't seem to make it to the final compiled version resulting in a seemingly invisible scrollbar (since the other classes still make it).

**Info**
- tailwind-scrollbar `v1.3.1`
- tailwindcss `v2.2.7` (with `mode: 'jit'`)
- postcss-cli `v8.3.1`.  
- Node `v14.12.0`  
- Windows.

**Comment by harvanchik** [comment]
Hoping this gets resolved.

**Comment by alixcan** [comment]
Is there any fix for this ?

**Comment by Wyn-Price** [comment]
@alixcan Are you able to try changing the `tailwind-scrollbar` version to `Wyn-Price/tailwind-scrollbar`, I believe it should fix this issue.

**Comment by alixcan** [comment]
I've just tried, it works perfectly.

**Comment by adoxography** [comment]
Should be fixed in `1.3.2`.
